java/src/jmri/jmrit/vsdecoder/swing/ManageLocationsFrame.java

```diff
@@ -33,6 +32,7 @@
 import jmri.jmrit.operations.setup.Setup;
 import jmri.jmrit.vsdecoder.VSDecoderManager;
 import jmri.jmrit.vsdecoder.listener.ListeningSpot;
+import jmri.jmrit.vsdecoder.swing.ManageLocationsTableModel;
```

Comment:
This import is redundant. You don't need to import from the same package. Sometimes I have seen that the IDE (for example NetBeans) has added these wrongly.

The PR looks good, but the `Static Analysis` CI check complains about it so it needs to be fixed before the PR is merged.

Reply:
Thank you. Now fixed. It was my mistake.